CMakeLists.txt

```diff
@@ -140,7 +144,34 @@ if(AVIF_LOCAL_LIBYUV)
         find_package(JPEG REQUIRED)
         set(LIBYUV_LIBRARY ${LIBYUV_LIBRARY} ${JPEG_LIBRARY})
     endif()
+    set(libyuv_FOUND TRUE)
```

Comment:
Add a message, similar to the one at line 161, to indicate that libyuv-based fast paths are enabled.

Note that the message at line 161 requires the `LIBYUV_VERSION` variable, which is not defined here, so take care if you copy that message here or move that message to the `if(libyuv_FOUND)` block at line 170.

Reply:
DOne